src/components/Modal/EditSpace.vue

```diff
@@ -91,6 +102,31 @@ watch(
         </div>
       </template>
     </template>
+    <div v-if="!showPicker" class="relative bg-skin-border h-[100px] -mb-[50px]" />
+    <SIUploadImage
+      v-if="!showPicker"
+      ref="avatarInput"
+      class="relative h-[80px] ml-4 group max-w-max cursor-pointer"
+      @image-uploaded="url => (form.avatar = url)"
+    >
+      <template #avatar="{ uploading, previewUrl }">
```

Comment:
uploading prop doesn't seem to do anything, I see no difference when image is being uploaded.